build_variables.bzl

```diff
@@ -887,8 +890,8 @@ libtorch_python_core_sources = [
     "torch/csrc/autograd/python_variable.cpp",
     "torch/csrc/autograd/python_variable_indexing.cpp",
     "torch/csrc/jit/backends/backend_init.cpp",
+    "torch/csrc/jit/codegen/cuda/instrumentation.cpp",
```

Comment:
@kevinstephano can we remove this? The include is duplicated above, and internally it's causing some symbols to be built twice (and then the build fails during linking because of duplicate symbols)

Reply:
Hmm, I think so.  That might explain why I was having to qualify it with `IS_CUDA`.  I didn't realize the file was not with the rest of our source files.  I'll give it a try!

Reply:
I removed the `instrumention.cpp` build line duplication.  I am hoping this fixes the issue.🤞